_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -57378,7 +57367,6 @@
 	},
 /obj/item/seeds/carrot,
 /obj/effect/turf_decal/stripes/line,
-/obj/item/seeds/cannabis/white,
```

Comment:
also booooooooooooooooooo

grinding for these things with RNG is a bitch without that one botany tech, and you know it. having much easier/less RNG-dependent access to the evolved weed variants was one of the things that made metastation's maint more distinct from the maintenance layouts of other stations, and it was a nice thing to be able to flex knowledge about.

okay, that was admittedly a much weaker argument than my other arguments, but I like maint weed, okay? I like it when people actually use the ghetto setups in maint for gimmicks and such instead of just ignoring them because they got nerfed into being practically just aesthetic!

Reply:
mutated plants are supposed to be grindy not to be handed out every round just cause it looks nice.

Reply:
but the weed mutations are especially grindy because there are like 4 of them from a single plant.